addons/web/static/src/notifications/notification.xml

```diff
@@ -3,24 +3,19 @@
 
     <div t-name="web.NotificationWowl" t-attf-class="o_notification {{className}}"
          role="alert" aria-live="assertive" aria-atomic="true" owl="1">
-        <t t-set="closeButton">
+        <div class="o_notification_body">
             <button type="button" class="close o_notification_close"
                 aria-label="Close"
                 t-on-click="notificationService.close(props.id)">
-                <span class="fa fa-close" aria-hidden="true"/>
+                <span class="d-inline" aria-hidden="true">×</span>
             </button>
-        </t>
-        <div t-if="props.title" class="o_notification_header">
-            <span t-attf-class="fa fa-2x mr-3 {{icon}} o_notification_icon" role="img"/>
-            <div class="mr-auto font-weight-bold o_notification_title" t-raw="props.title"/>
-            <t t-raw="closeButton"/>
-        </div>
-        <div class="o_notification_body">
-            <t t-if="!props.title" t-raw="closeButton"/>
+            <div t-if="props.title" class="font-weight-bold o_notification_title mb-1" t-raw="props.title"/>
+            <strong t-if="props.subtitle" t-raw="props.subtitle"
+                class="o_notification_subtitle mb-2 font-weight-normal text-muted"/>
             <div t-if="props.message" t-raw="props.message" class="mr-auto o_notification_content"/>
             <div t-if="props.buttons.length" class="mt-2 o_notification_buttons">
                 <button t-foreach="props.buttons" t-as="button" type="button"
-                        t-attf-class="btn btn-sm {{button.primary ? 'btn-primary' : 'btn-secondary'}}">
+                        t-attf-class="btn {{button.primary ? 'btn-primary' : 'btn-link'}}" t-on-click="button.onClick()">
```

Comment:
question for review: should we close he notif as soon as we click on a button ?
or should we let the developper decides entirely what he wants to do on click (as it is now) ?

Reply:
Seems good as it is right now